openjdk-17.yaml

```diff
@@ -89,49 +91,53 @@ pipeline:
   - runs: make jdk-image
 
   - runs: |
-      mkdir -p ${{targets.destdir}}/usr/lib/jvm/openjdk
-      cp -r ./build/linux-*-server-release/images/jdk/* ${{targets.destdir}}/usr/lib/jvm/openjdk
+      mkdir -p ${{targets.destdir}}/usr/lib/jvm/java-17-openjdk
+      cp -r ./build/linux-*-server-release/images/jdk/* ${{targets.destdir}}/usr/lib/jvm/java-17-openjdk
 
       mkdir -p ${{targets.destdir}}/usr/bin
-      ln -sf /usr/lib/jvm/openjdk/bin/java ${{targets.destdir}}/usr/bin/java
-      ln -sf /usr/lib/jvm/openjdk/bin/javac ${{targets.destdir}}/usr/bin/javac
+      ln -sf /usr/lib/jvm/java-17-openjdk/bin/java ${{targets.destdir}}/usr/bin/java
```

Comment:
It would be better to integrate this into the `java-common` work I did.  If you create a `-default-jvm` package, with a symlink `/usr/lib/jvm/default-jvm` -> `/usr/lib/jvm/java-17-openjdk`, then Java 17 will automatically take ownership of all of the Java-related symlinks when the `-default-jvm` package is installed.  See `java-gcj-compat` and `openjdk-7` for examples of how the `-default-jvm` stuff works.